index.html

```diff
@@ -2276,22 +2284,133 @@ <h3 id="applying">
       </section>
       <section>
         <h3>
+          <code>purpose</code> member
+        </h3>
+        <p>
+          The <dfn><code>purpose member</code></dfn> is an <a>unordered set of
+          unique space-separated tokens</a> that are <a>ASCII
+          case-insensitive</a>. The allowed values the <a>icon purposes</a>.
```

Comment:
The allowed values the <a>icon purposes</a>. I don't understand. Missing "are:"?